README.md

```diff
@@ -370,3 +370,75 @@ column.
 
 Note, when using this expression. the thing you are counting is tags, not events, so if you
 have 10 events, each of which has 10 tags, then a `count()` of `tags_key` will return 100.
+
+
+##  Migrations
+
+Snuba provides functionality to manage ClickHouse database migrations. Migrations
+in Snuba are organized into groups, which typically correspond with features / sets
+of related tables in Snuba. In some cases, groups can provide a mechanism to test
+experimental features in Snuba that should not be rolled out to most users yet.
+Migration groups can be defined as either optional or mandatory. Optional groups
```

Comment:
Is it possible to have two groups that have migrations on the same table? Or should groups be split by table as well?

Reply:
The language here is a little vague on this because I think the answer is generally no, but in some very rare cases maybe. The scenario where we may want this is if we are adding an experimental migration to a table and we don't want anyone else to run it. It would provide a mechanism to turn off the migration for everyone except ourselves. But as a general rule, sets of related tables should all be in the same group.